lang/en.js

```diff
@@ -66,6 +66,7 @@ export default {
 	"components.input-time-range.errorBadInput": "{startLabel} must be before {endLabel}",
 	"components.input-time-range.startTime": "Start Time",
 	"components.interactive.instructions": "Press Enter to interact, Escape to exit",
+	"components.link.open-in-new-window": "Opens in a new window.",
```

Comment:
This should be copied into all the other language files otherwise the validator is going to complain (see all its angry  comments below).